src/cdk/v2/destinations/koddi/data/ClicksConfig.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "sourceKeys": "properties.tracking_data",
+    "required": true,
+    "destKey": "trackingData"
+  },
+  {
+    "sourceKeys": "properties.rank",
+    "required": true,
+    "destKey": "rank"
+  },
+  {
+    "sourceKeys": "properties.beacon_issued",
+    "required": true,
+    "destKey": "beaconIssued"
+  },
+  {
+    "sourceKeys": "userId",
+    "sourceFromGenericMap": true,
+    "required": true,
+    "destKey": "userGuid"
+  },
+  {
+    "sourceKeys": "properties.test_version_override",
+    "destKey": "testVersionOverride"
+  },
+  {
+    "sourceKeys": "properties.destination_url",
```

Comment:
can' we get this from `page` object?

Reply:
@manish339k can you tell what is the definition or use case of destination url here?

Reply:
<img width="645" alt="Screenshot 2024-05-21 at 10 53 16 AM" src="https://github.com/rudderlabs/rudder-transformer/assets/144022547/f5363617-6e54-4a1d-ab7b-387f996361be">